neuraxle/steps/flow.py

```diff
@@ -380,7 +368,7 @@ class ChooseOneStepOf(FeatureUnion):
     """
 
     def __init__(self, steps, hyperparams=None):
-        FeatureUnion.__init__(self, steps)
+        FeatureUnion.__init__(self, steps, joiner=SelectNotEmptyJoiner())
```

Comment:
joiner=SelectANonEmptyDataContainer?

Reply:
I named it SelectNonEmptyDataInputs because this is what it does.